packages/@react-spectrum/s2/src/Accordion.tsx

```diff
@@ -70,3 +79,42 @@ export const Accordion = forwardRef(function Accordion(props: AccordionProps, re
     </DisclosureContext.Provider>
   );
 });
+
+export interface AccordionItemProps extends DisclosureProps {
+  /** The contents of the accordion, consisting of a AccordionItemTitle and AccordionItemPanel. */
+  children: ReactNode
+}
+/**
+ * A accordion item is a collapsible section of content. It is composed of a a header with a heading and trigger button, and a panel that contains the content.
+ */
+export const AccordionItem = forwardRef(function AccordionItem(props: AccordionItemProps, ref: DOMRef<HTMLDivElement>) {
+  return <Disclosure {...props} ref={ref} />;
+});
+
+export const AccordionItemContext = DisclosureContext;
```

Comment:
again, not sure if this should be re-exported or a separate context (and maybe implementation) should be made to avoid a breaking change